### ISSUE TITLE
Minor Sprite Corruption

### ISSUE THREAD

**Issue by Videodr0me** [MiSTer-devel/Arcade-MoonPatrol_MiSTer#29]
After playing 30 seconds or so, exactly at the same time as  the ufo sprites appear, there is some minor sprite corruption near the front wheel of the moon rover.  At least on my system its 100% reproducable. Strangely i never noticed that before, so it might have been introduced recently.

**Comment by Real-Dampfwalze** [comment]
Can confirm this.

**Comment by HorstKlein67** [comment]
Yes, when you play a while, every now and then, small moving lines appear on the screen, mostly when you and UFOs are shooting. Sometimes parts of UFOs are missing. It is sometimes confusing because you think there are additional bombs coming down.

**Comment by Real-Dampfwalze** [comment]
This sprite problem seems somehow to be fixed. Last time I played it did not show up.
So it remains only the missing solid background in the level set with the city in the background to be fixed.

**Comment by febs** [comment]
Is this still an issue or could this be closed as well? @Real-Dampfwalze . Thanks,

**Comment by Real-Dampfwalze** [comment]
In my opinion, this can be closed.
The background is fixed and the sprite problems too. 👍
